comictaggerlib/ctsettings/commandline.py

```diff
@@ -251,6 +251,30 @@ def register_commands(parser: settngs.Manager) -> None:
         help="List the available plugins.\n\n",
         file=False,
     )
+    parser.add_setting(
+        "--list-remote-plugins",
+        dest="command",
+        action="store_const",
+        const=Action.list_remote_plugins,
+        default=Action.gui,
+        help="List the available remote plugins for install/update.\n\n",
+        file=False,
+    )
+    parser.add_setting(
+        "--update-remote-plugins",
+        dest="command",
+        action="store_const",
+        const=Action.update_remote_plugins,
+        default=Action.gui,
+        help="Update all installed remote plugins.\n\n",
+        file=False,
+    )
+    parser.add_setting(
+        "--install-remote-plugin",
+        default="",
+        help="Install (or update) a remote plugin via its ID, eg. 'metronxml'. Use --list-remote-plugins for list.\n\n",
```

Comment:
Right now IDs are not unique across different plugin types and Archivers don't even have IDs, they just get added to the list to be tried.

Reply:
This would be the ID not from the plugin but the manifest.

Reply:
We need to not use id then because it's used for a different purpose everywhere else.
You could rename id to name and only reference name, this is more of a convenience thing so I'm not interested in making things needlessly convoluted.

Reply:
The two are now related since `plugin_finder.find_plugins()` is now used and the one ID in the manifest is checked against the plugin ID to make sure they match.

The manifest does have the `type` of plugin so the `id` and `type` could be checked.

Archivers will still require an `entry_name` which would be the ID? I guess the thing to do it take a builtin one and make it a plugin to check. 

Should external plugin archivers require an ID anyway?

Reply:
Testing an external archiver using the `entry_name` as a standin for ID looks to be reasonable?